src/NuGetGallery.Services/Configuration/FeatureFlagService.cs

```diff
@@ -37,6 +37,7 @@ public class FeatureFlagService : IFeatureFlagService
         private const string Get2FADismissFeedback = GalleryPrefix + "Get2FADismissFeedback";
         private const string UsabillaOnEveryPageFeatureName = GalleryPrefix + "UsabillaEveryPage";
         private const string PackageRenamesFeatureName = GalleryPrefix + "PackageRenames";
+        private const string UploadEmbeddedReadMeFeatureName = GalleryPrefix + "UploadEmbeddedReadMeFeatureName";
```

Comment:
Can we change the styling of readme everywhere from `ReadMe` to `Readme`? This is a single word, our previous usage was incorrect.

```suggestion
        private const string UploadEmbeddedReadMeFeatureName = GalleryPrefix + "UploadEmbeddedReadmeFeatureName";
```